.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels: 'type: bug'
```

Comment:
```suggestion
title: '[bug]: '
labels: 'type: bug'
```

Reply:
Isn't it redundant to have bug both in the title and label?

Reply:
yeah, to some extent but in my personal experience,  I always look at the title before labels and I think it helps me as a maintainer  to have a quicker idea about the issue if bug/feat/docs is in the issue title. 

Also there is other places where labels are not shown like preview links (like when you post it in discord) or link to it in another issue

Reply:
Fair point. I'll add them without the `:` to save a couple precious pixels